bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcPanel.scala

```diff
@@ -388,6 +393,14 @@ private class AnalysisMethodPanel extends GridBagPanel {
     case ValueChanged(_)                => publish(new SelectionChanged(this))
   }
 
+  def update() = {
+    // OCSADV-345: Don't allow users to change sky aperture for NIRI, F2 and GSAOI, this functionality has not been verified for these instruments.
+    Option(owner.getContextInstrumentDataObject).foreach { _.getType match {
+      case InstNIRI.SP_TYPE | Flamingos2.SP_TYPE | Gsaoi.SP_TYPE  => sky.enabled = false; skyLabel.enabled = true; sky.peer.setValue(1.0)
+      case _                                                      => sky.enabled = true;
+    }}
+  }
```

Comment:
I've discussed this with Andy and he prefers to disable the components over having them made invisible because like that people see which value is used for the calculations. We could maybe add a tooltip that gives an explanation for why this value is disabled for those three instruments.

Those values are not (yet) made persistent. Which is actually something Andy has a task for, however it might be pushing it a bit to add those to the observation for this release. For now, they are not backed by anything in the observation and are just used on-the-fly for the calculations.